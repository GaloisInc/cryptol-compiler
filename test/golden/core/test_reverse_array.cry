main = reverse [0x01:[8]..0x10]