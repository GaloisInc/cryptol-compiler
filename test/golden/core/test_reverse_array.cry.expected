[0x10, 0x0f, 0x0e, 0x0d, 0x0c, 0x0b, 0x0a, 0x09, 0x08, 0x07, 0x06, 0x05, 0x04, 0x03, 0x02, 0x01]
